<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>673</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Bitmap Image Embedder</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QCheckBox::indicator
 {
width: 20px;
height: 20px;
}
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCheckBox" name="bit7">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>590</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="bit6">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>590</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QCheckBox" name="bit5">
    <property name="geometry">
     <rect>
      <x>195</x>
      <y>590</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="bit4">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>590</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="bit3">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>590</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="bit2">
    <property name="geometry">
     <rect>
      <x>315</x>
      <y>590</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="bit1">
    <property name="geometry">
     <rect>
      <x>355</x>
      <y>590</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="bit0">
    <property name="geometry">
     <rect>
      <x>395</x>
      <y>590</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="retrieveBtn">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>290</y>
      <width>130</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Retrieve Secret</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>160</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Max Secret Size (B):</string>
    </property>
   </widget>
   <widget class="QLabel" name="fileSize">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>170</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="ImageView" name="mainImage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>591</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="dragMode">
     <enum>QGraphicsView::ScrollHandDrag</enum>
    </property>
    <property name="transformationAnchor">
     <enum>QGraphicsView::NoAnchor</enum>
    </property>
    <property name="resizeAnchor">
     <enum>QGraphicsView::NoAnchor</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="clearCarrierBtn">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>220</y>
      <width>130</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>590</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Bit Postions:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>401</x>
      <y>610</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>610</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>610</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>610</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>610</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>610</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>610</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>610</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Image:</string>
    </property>
   </widget>
   <widget class="QLabel" name="displayImgPath">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>661</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>869</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionClear_Image"/>
    <addaction name="actionRetrieve_Secret"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHow_to_Use"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionClear_Image">
   <property name="text">
    <string>Clear Image</string>
   </property>
  </action>
  <action name="actionRetrieve_Secret">
   <property name="text">
    <string>Retrieve Secret</string>
   </property>
  </action>
  <action name="actionHow_to_Use">
   <property name="text">
    <string>How to Use</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ImageView</class>
   <extends>QGraphicsView</extends>
   <header>imageView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
